Ednpoints.txt

```diff
@@ -0,0 +1,132 @@
+The same routes for users, chefs and delivery
+POST: /auth/register - registration
```

Comment:
I don't think it should be a part of auth group
maybe consider something like:
POST /users
POST /admins
POST /deliveries
POST /chefs

Reply:
You're right, it's logical